Procfile

```diff
@@ -0,0 +1,4 @@
+# Run Rails & Webpack concurrently
+# Example file from webpack-rails gem
+rails: bundle exec rails server
```

Comment:
should be possible based on https://github.com/webpack/webpack-dev-server/pull/263